modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -67,14 +77,10 @@ ifndef::openshift-dedicated,openshift-rosa[]
 3.
 
 * Does not support offline snapshots or resize. Volume must be attached to a running pod.
+
+4.
```

Comment:
The numbering doesn't look correct here in the preview. I _think_ you need `^1^` etc below the table for it to render correctly.